package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "record_commit_hash_and_build_time": "now=$(date -u \"+%Y-%m-%d %H:%M:%S\") && last_commit=$(git rev-parse HEAD) && echo \"{\\\"commit\\\": \\\"$last_commit\\\", \\\"build_time\\\": \\\"$now\\\"}\" > public/version.json",
+    "prebuild": "yarn record_commit_hash_and_build_time",
```

Comment:
what is this used for? Is it necessary for developers to interact with this application?

Reply:
@nearken It's convenient to be able to browse example.com/version.json and see this kind of info.